ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -31,9 +34,10 @@ def __init__(self, *args, **kwargs):
             raise errors.PymqiException("You need to install pymqi: {}".format(pymqiException))
 
         self.queue_metric_collector = QueueMetricCollector(
-            self.config, self.service_check, self.warning, self.send_metric, self.log
+            self.config, self.service_check, self.warning, self.send_metric, self.send_metrics_from_properties, self.log
         )
         self.channel_metric_collector = ChannelMetricCollector(self.config, self.service_check, self.gauge, self.log)
+        self.stats_collector = StatsCollector(self.config, self.send_metrics_from_properties, self.log)
```

Comment:
note: some refactoring is needed to avoid passing so many methods as parameters (will be done in another PR)